css/index.css

```diff
@@ -1,9 +1,223 @@
-/* Example of @import use
-This imports the `reset.css` file */
-@import "./reset.css";
-
-/* For testing purposes, delete this code */
-body {
-  font-size: max(20vw, 400px);
-  width: min(20vw, 400px);
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.primary-bg {
+  color: white;
+  background-color: yellow;
+}
+
+.edit-me {
+  background-color: #1f2937;
```

Comment:
Right here this is what I mean by using HEX color, let try and make all our colors something like this